example.env

```diff
@@ -0,0 +1,4 @@
+# Copy the contents of this file into a new file called .env.
+# Sets the port to 3000 if not otherwise defined on server/index.js
+
+PORT=3000
```

Comment:
The .env file is not set up yet.

Reply:
Or did you set it up locally but added it to gitignore?

Reply:
I'm going to close this, update the file to include the port pointing at the env file, and repull